nbextensions/usability/hinterland/hinterland.js

```diff
@@ -69,8 +70,13 @@ define(function (require, exports, module) {
 						}, cur);
 
 						if (pre_cursor !== '' && !ignore_re.test(pre_cursor)) {
-							completer.startCompletion();
-							completer.autopick = false;
+							if (pre_cursor === '(') {
+								cell.tooltip.request(cell);
```

Comment:
If you find it, please also fix upstream :-)  -> https://github.com/jupyter/notebook/issues/1296